tests/make-backup-and-restore-iso/runtest.sh

```diff
@@ -73,13 +73,15 @@ rlJournalStart
         rlPhaseEnd
 
         rlPhaseStartSetup "Create /etc/rear/local.conf"
-            rlFileBackup "/etc/rear/local.conf"
+            rlFileBackup "$REAR_CONFIG"
             rlRun -l "echo 'OUTPUT=USB
 BACKUP=NETFS
 BACKUP_URL=usb://$REAR_LABEL_PATH
-USER_INPUT_TIMEOUT=10' | tee /etc/rear/local.conf" \
+USER_INPUT_TIMEOUT=10
+ISO_DEFAULT=automatic
+ISO_RECOVER_MODE=unattended' | tee $REAR_CONFIG" \
```

Comment:
These changes are included in 4cafe306d7e7c0f7e74dc90648295e0d05cb8987 but do not seem to be related to it at all.